### PR TITLE
New Customizer section "Month View"

### PR DESCRIPTION
**Pull request by Camwyn**
[TEC-3836]
[TEC-3840]
[TEC-3841]

Requires https://github.com/the-events-calendar/tribe-common/pull/1580
📷 https://d.pr/i/D0GI1o

[TEC-3836]: https://theeventscalendar.atlassian.net/browse/TEC-3836

[TEC-3840]: https://theeventscalendar.atlassian.net/browse/TEC-3840
[TEC-3841]: https://theeventscalendar.atlassian.net/browse/TEC-3841

### CODE REVIEW

File: src/Tribe/Views/V2/Customizer/Section/Month_View.php [hunk 1]
```diff
@@ -0,0 +1,446 @@
...
+	 */
+	public function get_css_template( $template ) {
+		if ( ! tribe_events_views_v2_is_enabled() ) {
+			return $template;
+		}
+
+		// These allow us to continue to _not_ target the shortcode.
+		$apply_to_shortcode = apply_filters( 'tribe_customizer_should_print_shortcode_customizer_styles', false );
+		$tribe_events = $apply_to_shortcode ? '.tribe-events' : '.tribe-events:not( .tribe-events-view--shortcode )';
+
```
Comment (borkweb) [inline src/Tribe/Views/V2/Customizer/Section/Month_View.php:368]: Superfluous linebreak removal:

```suggestion
```

File: src/Tribe/Views/V2/Customizer/Section/Month_View.php [hunk 2]
```diff
@@ -0,0 +1,446 @@
...
+
+		if ( $this->should_include_setting( 'grid_lines_color' ) ) {
+			$template .= "
+			$tribe_events.tribe-common--breakpoint-medium .tribe-events-calendar-month__body,
+			$tribe_events.tribe-common--breakpoint-medium .tribe-events-calendar-month__day,
+			$tribe_events.tribe-common--breakpoint-medium .tribe-events-calendar-month__week {
+				border-color: <%= month_view.grid_lines_color %>;
+			}
```
Comment (borkweb) [inline src/Tribe/Views/V2/Customizer/Section/Month_View.php:376]:
```suggestion
				$tribe_events.tribe-common--breakpoint-medium .tribe-events-calendar-month__body,
				$tribe_events.tribe-common--breakpoint-medium .tribe-events-calendar-month__day,
				$tribe_events.tribe-common--breakpoint-medium .tribe-events-calendar-month__week {
					border-color: <%= month_view.grid_lines_color %>;
				}
```

File: src/Tribe/Views/V2/Customizer/Section/Month_View.php [hunk 3]
```diff
@@ -0,0 +1,446 @@
...
+			";
+		}
+
+		if ( $this->should_include_setting( 'grid_hover_color' ) ) {
+			$template .= "
+			$tribe_events.tribe-common--breakpoint-medium .tribe-events-calendar-month__day:hover::after {
+				background-color: <%= month_view.grid_hover_color %>;
+			}
```
Comment (borkweb) [inline src/Tribe/Views/V2/Customizer/Section/Month_View.php:384]:
```suggestion
				$tribe_events.tribe-common--breakpoint-medium .tribe-events-calendar-month__day:hover::after {
					background-color: <%= month_view.grid_hover_color %>;
				}
```

File: src/Tribe/Views/V2/Customizer/Section/Month_View.php [hunk 4]
```diff
@@ -0,0 +1,446 @@
...
+			";
+		}
+
+		if ( $this->should_include_setting( 'grid_background_color_choice' ) ) {
+			if ( $this->should_include_setting( 'grid_background_color' ) ) {
+				$template .="
+				.tribe-events-calendar-month__body {
+					background-color: <%= month_view.grid_background_color %>;
+				}
```
Comment (borkweb) [inline src/Tribe/Views/V2/Customizer/Section/Month_View.php:393]:
```suggestion
					.tribe-events-calendar-month__body {
						background-color: <%= month_view.grid_background_color %>;
					}
```

File: src/Tribe/Views/V2/Customizer/Section/Month_View.php [hunk 5]
```diff
@@ -0,0 +1,446 @@
...
+				";
+			}
+		} else {
+			if ( $this->should_include_setting( 'tooltip_background_color' ) ) {
+				$template .="
+				.tooltipster-base.tribe-events-tooltip-theme,
+				.tooltipster-base.tribe-events-tooltip-theme--hover {
+					background-color: <%= global_elements.background_color %>;
+				}
```
Comment (borkweb) [inline src/Tribe/Views/V2/Customizer/Section/Month_View.php:402]:
```suggestion
					.tooltipster-base.tribe-events-tooltip-theme,
					.tooltipster-base.tribe-events-tooltip-theme--hover {
						background-color: <%= global_elements.background_color %>;
					}
```

File: src/Tribe/Views/V2/Customizer/Section/Month_View.php [hunk 6]
```diff
@@ -0,0 +1,446 @@
...
+				";
+			}
+		}
+
+		if ( $this->should_include_setting( 'days_of_week_color' )  ) {
+			$template .="
+			$tribe_events .tribe-events-calendar-month__header-column-title {
+				color: <%= month_view.days_of_week_color %>;
+			}
```
Comment (borkweb) [inline src/Tribe/Views/V2/Customizer/Section/Month_View.php:411]:
```suggestion
				$tribe_events .tribe-events-calendar-month__header-column-title {
					color: <%= month_view.days_of_week_color %>;
				}
```

File: src/Tribe/Views/V2/Customizer/Section/Month_View.php [hunk 7]
```diff
@@ -0,0 +1,446 @@
...
+			";
+		}
+
+		if ( $this->should_include_setting( 'date_marker_color' )  ) {
+			$template .="
+			.tribe-events-calendar-month__day-date.tribe-common-h4,
+			$tribe_events .tribe-events-calendar-month__day-date-link {
+				color: <%= month_view.date_marker_color %>;
+			}
```
Comment (borkweb) [inline src/Tribe/Views/V2/Customizer/Section/Month_View.php:420]:
```suggestion
				.tribe-events-calendar-month__day-date.tribe-common-h4,
				$tribe_events .tribe-events-calendar-month__day-date-link {
					color: <%= month_view.date_marker_color %>;
				}
```

File: src/Tribe/Views/V2/Customizer/Section/Month_View.php [hunk 8]
```diff
@@ -0,0 +1,446 @@
...
+
+				$template .="
+				$tribe_events .tribe-events-calendar-month__multiday-event-bar-inner {
+					background-color: $bar_color;
+				}
+
+				$tribe_events .tribe-events-calendar-month__multiday-event-bar-inner--hover,
+				$tribe_events .tribe-events-calendar-month__multiday-event-bar-inner--focus {
+					background-color: $bar_color_hover;
+				}
```
Comment (borkweb) [inline src/Tribe/Views/V2/Customizer/Section/Month_View.php:438]:
```suggestion
					$tribe_events .tribe-events-calendar-month__multiday-event-bar-inner {
						background-color: $bar_color;
					}

					$tribe_events .tribe-events-calendar-month__multiday-event-bar-inner--hover,
					$tribe_events .tribe-events-calendar-month__multiday-event-bar-inner--focus {
						background-color: $bar_color_hover;
					}
```

**Review by borkweb — CHANGES_REQUESTED** [review]

**Review by borkweb — COMMENTED** [review]
Also, please name the PR with an approachable phrase.

File: readme.txt [hunk 9]
```diff
@@ -223,6 +223,9 @@ Remember to always make a backup of your database and files before updating!
 
 = [5.6.1] TBD =
 
+<<<<<<< HEAD
```
Comment (borkweb) [inline readme.txt:226]: Fix this merge conflict and this is coming along nicely!

**Review by borkweb — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 9 above)
Comment (Camwyn) [reply readme.txt:226]: 🤦🏼

**Review by borkweb — APPROVED** [review]
:+1:

**Pull request merged**